.gitignore

```diff
@@ -1,2 +1,6 @@
 *.swp
 local.toml
+
+# editor files
```

Comment:
And this too just in case:

`.vscode/`

Reply:
Added :)